addons/website/static/src/snippets/s_rating/001.scss

```
@@ -13,3 +13,32 @@
         }
     }
 }
+
+.s_rating_inactive_icons {
+    margin-left:-0.25em;
+    .fa {
+        margin-left: 0.5em;
+    }
+    .s_extra_margin {
+        margin-left: 0.75em;
+    }
+    .s_idle_margin {
+        margin-left: 0.25em;
+    }
+    .s_idle_extra_margin {
+        margin-left: 0.5em;
+    }
+}
+
+.s_rating_active_icons {
+    .fa {
+        margin-left: 0.5em;
+    }
+    .s_idle_margin {
+        margin-left: 0.25em;
+    }
+    .s_idle_extra_margin {
+        margin-left: 0.5em;
+    }
+}
```

And with that we wouldn't need the classes on the <i/>
```suggestion 
.s_rating_active_icons,.s_rating_inactive_icons {
    margin: 0!important;
    padding: 0!important;
    .fa {
        margin: 0 0 0 .5em!important;
    }
}
```